website/src/examples/bundle/app.html

```diff
@@ -5,7 +5,7 @@
 <script src="/uppy/uppy.min.js"></script>
 <script>
   var uppy = Uppy.Core({debug: true});
-  uppy.use(Uppy.Tus, {endpoint: '//master.tus.io/files/'});
+  uppy.use(Uppy.Tus, {endpoint: 'https://master.tus.io/files/'});
```

Comment:
Question: what is `/examples/bundle/` about, we don't seem to have a link to it on a site anywhere, is it for some local testing? It works, but we need to add `{ autoProceed: true }`, and actually attach ProgressBar somewhere.

Reply:
Good catch, I think it got superseded by https://uppy.io/examples/i18n/, I’ll remove it.